index.json

```diff
@@ -1,7 +1,7 @@
 {
   "wows": {
-    "index": "",
-    "disabled": false
+    "index": "/wows/data.json",
+    "disabled": true
```

Comment:
Plz, no disable 😂

Reply:
CSGO is disabled. wasn't sure whether it should be true or false